src/app/components/SVG/SVG.tsx

```diff
@@ -0,0 +1,50 @@
+import { classNames } from '../../utils/styles';
+import classes from './SVG.module.css';
+
+export type SVGProps = {
+  children: JSX.Element;
```

Comment:
There are other types possible like arrays or strings. The correct type is:

```suggestion
  children: React.ReactNode;
```

Reply:
thanks, Typescript didn´t complain 😉